sdk/identity/Azure.Identity/perf/CacheReadPerf.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Identity;
+using Azure.Test.Perf;
+
+namespace Azure.Template.Perf
+{
+    public class CacheReadPerf : PerfTest<CountOptions>
```

Comment:
This seems closer to a microbenchmark. Can we even see improvements when running under the perf harness?

Reply:
This is more of a baseline for the `CacheReadWithDiskPersistencePerf` test. But it is also useful as a regression test for MSAL's in memory cache implementation.